sources/packages/web/src/components/institutions/active-application/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <body-header
+    title="Applications"
+    :recordsCount="applications.results?.length"
+    class="m-1 mb-8"
```

Comment:
`mb-8` is a pretty big space. Why was it needed?
Are those styles not needed in the component?